test/support/async.ex

```diff
@@ -14,7 +15,7 @@ defmodule Support.Async do
              do: block
            ) do
     quote do
-      id = :crypto.strong_rand_bytes(12) |> Base.encode16()
+      id = :erlang.unique_integer([:positive])
```

Comment:
Although not a place where performance matters, I believe it to be more readable